templates/single-meetings.php

```diff
@@ -359,8 +359,35 @@ class="btn btn-default btn-block contact-phone">
                                     <?php
                                 } ?>
                                 <li class="list-group-item list-group-item-updated">
-                                    <?php _e('Updated', '12-step-meeting-list') ?>
-                                    <?php the_modified_date() ?>
+                                    <p class="meeting-updated">
+                                        <?php _e('Updated', '12-step-meeting-list') ?>
+                                        <?php the_modified_date() ?>
+                                    </p>
+                                    <?php
+                                    if (!empty($meeting->{'data_source'})) {
```

Comment:
i think we need some intro text here potentially like "This meeting listing is provided by:"

let's show the email address - this is more for administrative purposes

the feedback email form should either be hidden and we add something here, or it should send to the entity's feedback_email addresses if present

minor: could we use the `$meeting->data_source` pattern for this and the other vars?

minor: i would like to steer away from using `<br>` tags - could this be Ps or a UL?

Reply:
Adding "This meeting listing is provided by:" only concern it will show up everywhere if we display for local _and_ imported meetings, and it just adds that one more line to what your eye is scanning for
![image](https://github.com/user-attachments/assets/181a9d0c-1da1-4bce-905a-7f3bddd4e221)

Look correct to you? Should it be smaller?

Reply:
> or it should send to the entity's feedback_email addresses if present

Honestly this raises questions I want to talk about, cause it'll be a mess typing it out. Don't feel equipped with enough history on this tool to answer them myself.

Reply:
hm, yeah, i'm not so sure. perhaps less is more?

my reason for wanting to add it is that we want to make it clear what this section is - some users might get confused and think it is meeting information.

the purpose of this section is to be like this section in the app:
![Screenshot 2024-09-24 at 3 11 42 PM](https://github.com/user-attachments/assets/1c4dcfb7-dc2d-4812-ad8c-c4bcf47fba76)

if we could replicate the headline style for the other sections for the entity name, and make the url and phone buttons like in this pic:

<img width="330" alt="Screenshot 2024-09-24 at 3 14 05 PM" src="https://github.com/user-attachments/assets/0ea076c4-31b8-4acb-ada6-19a98f40154a">

we might try the `This meeting listing is provided by:` above the headline in a `<small>`?

Reply:
here's what it currently looks like in [this PR for TSML UI](https://github.com/code4recovery/tsml-ui/pull/416):

![image](https://github.com/user-attachments/assets/eb5fd843-cd10-4611-adfb-9ee60df1122e)

not that it needs to match per se but just for context